tests/src/actionContainers/Swift3ActionContainerTests.scala

```diff
@@ -98,16 +103,16 @@ class Swift3ActionContainerTests extends SwiftActionContainerTests {
 
         checkStreams(out, err, {
             case (o, e) =>
-                o shouldBe empty
+                //o shouldBe empty
```

Comment:
The swift package manager likes to print out what its doing and there doesn't seem to be a way to quiet it.